NEWS.md

```diff
@@ -10,3 +10,9 @@ CRUD interface for:
 
 ## 1.0.3 Release
 * resolved missing release artifacts
+
+## 2.0.0 Release
```

Comment:
If you look at the other repos I think they're typically organized so that the newest release is at the top, not bottom.